### PR TITLE
#781: Renamed AtlasUI to ZalandoCommerceUI

### PR DESCRIPTION
**Pull request by bonkey**
#781: Renamed AtlasUI to ZalandoCommerceUI

### CODE REVIEW

**Comment by bonkey** [comment]
Still few leftovers, will cleanup quickly

File: Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements [hunk 1]
```diff
@@ -3,8 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>keychain-access-groups</key>
-	<array>
-		<string>$(AppIdentifierPrefix)com.zalando.AtlasUITestsHostsApp</string>
```
Comment (hani-ibrahim) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements]: This is not used for the tests? idk

**Comment by hani-ibrahim** [comment]
👍

File: Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements (hunk 1 above)
Comment (bonkey) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements]: I pressed "fix" ;-) Anyway, keychain tests could be tricky, I saw them some time not raising some errors, and then back to normal... pulling on a list

Comment (bonkey) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements]: I mean showing some warnings about not working keychain. anyway will check.

Comment (hani-ibrahim) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements]: well, if the tests are passing then it's fine

Comment (bonkey) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements]: Hahaha, so lets just surround all tests with

```swift
        expect { 
        }.toNot(throwError(NeverThrownError))
```

Comment (bonkey) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUITestsHostsApp/ZalandoCommerceUITestsHostsApp.entitlements]: I mean, tests could give false safety, like not verified backups.

or just https://twitter.com/thepracticaldev/status/687672086152753152

**Pull request merged**